chainer/functions/normalization/batch_normalization.py

```diff
@@ -485,13 +478,10 @@ def forward(self, inputs):
                 expand_dim = True
                 x = x[:, :, None, None]
 
-            gamma = gamma[expander]
-            beta = beta[expander]
-            W = numpy.concatenate((gamma, beta), axis=0).reshape((2, -1))
-
             y, = intel64.ideep.batchNormalization.Forward(
                 intel64.ideep.array(x),
-                intel64.ideep.array(W),
+                intel64.ideep.array(gamma),
```

Comment:
* Does this PR require `ideep4py>=2.0.0`? I tried this PR with `ideep4py-2.0.0b1.post1-cp36-cp36m-manylinux1_x86_64.whl` on PyPI, but python crashed with `illegal hardware instruction (core dumped)` when I run tests. Could you please confirm?
* Will iDeep4py 2.0 break API compatibility with iDeep4py 1.x?

Reply:
yes, this PR require ideep4py>=2.0.0. For the illegal hardware instruction, it is caused by iDeep2.0, ideep4py-2.0.0b1.post1-cp36-cp36m-manylinux1_x86_64.whl is for avx512, ideep4py_avx2-2.0.0b1.post1-cp36-cp36m-manylinux1_x86_64.whl is for avx2. If your device is with avx2, please install ideep4py_avx2-2.0.0b1.post1-cp36-cp36m-manylinux1_x86_64.whl. Thanks.

Reply:
We have uploaded ideep4py2.0.0, which supports both avx2 and avx512

Reply:
Thank you, that's nice.